diesel_tests/tests/debug/mod.rs

```diff
@@ -1,4 +1,5 @@
 use crate::schema::TestBackend;
+use diesel::upsert::DecoratableTarget;
```

Comment:
To resolve this to make it so `use diesel::*` also pulls in the `DecoratableTarget`, you should add the trait as a public reexport in `lib.rs` of Diesel in the inline `prelude` module.

See this for more details: https://github.com/diesel-rs/diesel/blob/master/diesel/src/lib.rs#L308

Reply:
aaah, makes sense, thanks!